ddtrace/tracer/tracer.go

```diff
@@ -161,6 +161,8 @@ func Start(opts ...StartOption) {
 	cfg.ServiceName = t.config.serviceName
 	if err := t.startRemoteConfig(cfg); err != nil {
 		log.Warn("Remote config startup error: %s", err)
+	} else {
+		log.Info("Remote config started succesffully")
 	}
```

Comment:
Let's remove this log or make it debug - it's not helpful for us right now and it might be a surprise for other users of RC.

We will review all logs we emit from Go DI before releasing to customers, we need to be sure that we have all the information needed to troubleshoot customer issues.

Typo - succesffully

Reply:
i'll remove for now, I agree that we should review all logs before we release.